src/api/Authentication.js

```diff
@@ -0,0 +1,24 @@
+export class Authentication{
+  constructor(){}
+
+  static validityChecker(email, password){
+    const validPassword = Authentication.passwordChecker(password);
+    const validEmail = Authentication.emailChecker(email);
+    if(validEmail && validPassword) {
+      localforage.setItem('authenication', 'Successful');
+    }
+    return 'Successful';
```

Comment:
***Code Quality: Nitpick***
Is this not suppose to return error if the above if condition fails?